Dockerfile

```diff
@@ -9,6 +9,7 @@
 
 # envoy-binary pulls in the latest Envoy binary, as Envoy don't publish
 # prebuilt binaries in any other form.
+ARG GOLANG_VERSION
```

Comment:
~ Is this needed here?

Reply:
It needed to be added back.

Reply:
Ah, I think multi-stage needs it declared up top https://stackoverflow.com/questions/53681522/share-variable-in-multi-stage-dockerfile-arg-before-from-not-substituted